aiera.ai.vector.db.connectError=${0}数据库连接失败